File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-05/non-plagiarized/09/t05.kt
package IR

import java.util.Scanner

fun main() {
    val sc = Scanner(System.`in`)
    print("Enter an integer: ")
    val num = sc.nextInt()
    reverse(num)
}

fun reverse(number: Int) {
    val tmpNumber = number.toString()
    for (i in tmpNumber.length - 1 downTo 0) {
        print(tmpNumber[i])
    }
    println()
}

File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-05/non-plagiarized/09/t05.ts
import { Scanner } from 'readline-sync';

function reverse(number: number): void {
  const tmpNumber: string = number.toString();
  let result: string = '';
  for (let i: number = tmpNumber.length - 1; i >= 0; i--) {
    result += tmpNumber.charAt(i);
  }
  console.log(result);
}

function main(): void {
  const sc: Scanner = new Scanner();
  console.log('Enter an integer: ');
  const num: number = sc.nextInt();
  reverse(num);
}

main();

